node/prepublish.js

```diff
@@ -43,15 +43,14 @@ var repoUtils = require('./repoUtils');
 // Use npm.md as the README for the package.
 var absGitRepoPath = path.resolve(path.join(__dirname, '..', '..', '..'));
 var readmePath = path.join(absGitRepoPath, 'README.md');
-var readmeBackupPath = readmePath + '.orig';
+var readmeBackupPath = 'ORIGINAL.md';
```

Comment:
The only minor issue I have with this is that, with this update, the temporary path is no longer adhering to the repo path—it'll dump the file wherever you're running from, which may not be desirable.  Should we back it up to the repo?  Or an explicit temp dir, maybe?